simulator/scheduler/storereflector/storereflector.go

```diff
@@ -57,11 +57,22 @@ func (s *reflector) ResisterResultSavingToInformer(client clientset.Interface, s
 	informerFactory := scheduler.NewInformerFactory(client, 0)
 	// Reflector adds scheduling results when pod is updating.
 	// This is because Extenders doesn't have any phase to hook scheduling finished. (both successfully and non-successfully)
-	_, err := informerFactory.Core().V1().Pods().Informer().AddEventHandler(
-		cache.ResourceEventHandlerFuncs{
+	_, err := informerFactory.Core().V1().Pods().Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: func(obj interface{}) bool {
+			if _, ok := obj.(cache.DeletedFinalStateUnknown); ok {
```

Comment:
```suggestion
            if obj == nil {
                klog.V(5).Info("Filtering nil object")
                return false
            }

			if _, ok := obj.(cache.DeletedFinalStateUnknown); ok {
```

Reply:
Should we directly remove the judgment of cache.DeletedFinalStateUnknown?